url-updater/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.devonfw.tools.ide.dev</groupId>
+    <artifactId>devonfw-ide</artifactId>
+    <version>dev-SNAPSHOT</version>
+  </parent>
+  <groupId>com.devonfw.tools.ide</groupId>
+  <artifactId>devonfw-ide-url-updater</artifactId>
+  <packaging>jar</packaging>
+  <version>${revision}</version>
+  <name>${project.artifactId}</name>
+  <description>Code for maintenance of download urls of IDEs tools (Eclipse, Docker, etc.).</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>javax.json</groupId>
+      <artifactId>javax.json-api</artifactId>
+      <version>1.1.4</version>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish</groupId>
+      <artifactId>javax.json</artifactId>
+      <version>1.1.4</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifest>
+              <addClasspath>true</addClasspath>
+              <mainClass>com.devonfw.tools.ide.url-updater.Url-updater</mainClass>
```

Comment:
A Java class shall never have dashes in its name. Java-packages can not have dashes (will cause compile error AFAIK - did this change in newer Java version?).

Reply:
In the last commit I now used camel case resulting in the folder name "urlUpdater" instead of "url-updater". Still, I would be open to other ideas or further proposals.

Reply:
That's completely right. The mainClass was nonexistent anyway. Now I temporarily choose UrlRepository as main class, but maybe I should just leave this out for now, as we don't have a main class to start code execution yet?

Reply:
In general better leave such things out until they are really needed. However, not preventing a merge to get collaboration started...